outputs.tf

```diff
@@ -55,8 +55,8 @@ output "subnets_private_access" {
 }
 
 output "subnets_flow_logs" {
-  value       = [for network in module.subnets.subnets : network.enable_flow_logs]
-  description = "Whether the subnets will have VPC flow logs enabled"
+  value       = [for network in module.subnets.subnets : network.log_config]
```

Comment:
This is a breaking change. Instead I think we should continue to return binary true/false of whether it's active.

Reply:
Since 3.x.x removed enable_flow_logs arg, to keep the same interface I think we can check if log config object exists set true, if empty list set false. Does this approach work?

Reply:
@morgante I have switched back to returning binary t/f